app/controllers/agent/ManageAgentController.scala

```diff
@@ -75,6 +82,51 @@ class ManageAgentController @Inject()(
     }
   }
 
+  def manageAgentProperties(
+        agentCode: Long,
+        params: GetPropertyLinksParameters,
+        pagination: PaginationParameters): Action[AnyContent] = authenticated.async { implicit request =>
```

Comment:
pagination is not used

Reply:
Last week discussions with Karren, she wanted gohead with Search Functionality on Manage Agents but she chnaged her mind after yesterday our chat.

She wanted to display Simple Agents Property Details without "Search Functionality".

She will let me know soon when she wants swithcing between "Simple Agents" page and  "Search Functionality"

I will apply using Pagination and Filtering parameters when "Search Functionality" in play